Add Prague 2020 website (#1251)
writethedocs/www: docs/conf/prague/2020/schedule.rst

    :depth: 1
    :backlinks: none

> {% if flaghashike or flaghasboat %}

Reviewer: For some reason I can't insert a suggestion directly to the boat ride text, but we need to match it with what it says on the home page of the conference (i.e. "We invite you to do a boat ride or walk on your own)

Author: Done.

> The Writing Day and Welcome Reception will be held at the **{{about.venue}}**.

Reviewer: ```suggestion
The Writing Day and Welcome Reception will be held **{{about.venue}}**.
```

Reviewer: Fix this everywhere on the page, otherwise it breaks grammar

{% if flaghasfood %}

*Breakfast and lunch will be provided, as well as snacks and drinks all day.*

{% endif %}

.. contents::
    :local:
    :backlinks: none

Writing Day
~~~~~~~~~~~

> * **Where**: {{about.venue}}, {{about.unconference}}

Reviewer: ```suggestion
* **Where**: {{about.venue}}
```

> * **When**: **9:00-17:00 PST**

Reviewer: ```suggestion
* **When**: **9:00-17:00 CET**
```

  A detailed schedule will be announced soon.
{% endif %}

{% if flaghasfood %}

Reception
~~~~~~~~~

We encourage everyone to drop by on Sunday evening for the conference reception.
We're hoping to have some fun activities planned for the evening online.

* **Where**: {{about.venue}}, {{about.mainroom}}

> * **When**: **17:00-20:00 PST**

Reviewer: ```suggestion
* **When**: **17:00-20:00 CET**
```

{% endif %}

> Monday, August 10

Reviewer: ```suggestion
Monday, October 19
```

-----------------

.. contents::
   :local:
   :backlinks: none

This is the main event! Hear from lots of interesting folks about all things documentation.

> We will have talks all day, and an unconference session running in parallel in the afternoon.

Reviewer: ```suggestion
We will have talks all day, and unconference sessions running in parallel.
```

{% if flaghasfood %}

*Breakfast and lunch will be provided, as well as snacks and drinks all day.*

{% endif %}

Conference Talks
~~~~~~~~~~~~~~~~

> * **Where**: {{about.venue}}, {{about.mainroom}}

Reviewer: ```suggestion
* **Where**: {{about.venue}}
```

> * **When**: **10:00-18:00 PST**

Reviewer: ```suggestion
* **When**: **10:00-18:00 CET**
```

The unconference sessions run in parallel to the main conference talks.

* **Where**: {{about.venue}}, {{about.unconfroom}}

> * **When**: **10:40-18:00 PST**

Reviewer: ```suggestion
* **When**: **10:40-18:00 CET**
```

* **Details**: :doc:`/conf/{{shortcode}}/{{year}}/unconference`

Monday Night Social
~~~~~~~~~~~~~~~~~~~

The official Write the Docs social!
Further details will be announced later,
but expect some music and games,

> and bring your favorite beverage along to the computer :)

Reviewer: ```suggestion
and bring your favorite beverage to your computer :)
```

> Tuesday, August 11

Reviewer: ```suggestion
Tuesday, October 20
```

------------------

.. contents::
   :local:
   :backlinks: none

And the conference goes on!

{% if flaghasfood %}

*Breakfast and lunch will be provided, as well as snacks and drinks all day.*

{% endif %}

Conference Talks
~~~~~~~~~~~~~~~~

> * **Where**: {{about.venue}}, {{about.mainroom}}

Reviewer: ```suggestion
* **Where**: {{about.venue}}
```

> * **When**: **10:00-16:30 PST**

Reviewer: ```suggestion
* **When**: **10:00-16:30 CET**
```

We'll be holding a job fair on Tuesday morning!

* **Where**: {{about.venue}}, {{about.unconfroom}}

> * **When**: **10:30-11:50 PST**

Reviewer: ```suggestion
* **When**: **10:30-11:50 CET**
```

Reviewer: Maybe we can let this run longer? There are no space restrictions :)

> * **When**: **10:30-11:50 {{ tz }}**

Reviewer 2: We should backport this change to Portland if we haven't already 👍

Reviewer 3: It doesn't help us with 2:00 PM / 14:00 issue though. Is there an 'easy' format_this_according_to_conf_location function we can use?

Reviewer 2: Not that I know of off hand.

Reviewer: Going to resolve it for this PR and throw a card in the Global Trello board to handle timezones

The unconference sessions run in parallel to the main conference talks.

* **Where**: {{about.venue}}, {{about.unconfroom}}

> * **When**: **12:10-15:30 PST**

Reviewer: ```suggestion
* **When**: **12:10-15:30 CET**
```
